packages/preset-theme-default/package.json

```diff
@@ -49,5 +49,9 @@
     "@types/fs-extra": "^9.0.13",
     "@unocss/core": "^0.45.30",
     "fs-extra": "^11.0.0"
+  },
+  "peerDependencies": {
+    "@unocss/core": "^0.45.30",
+    "@anu-vue/preset": "^0.11.0"
```

Comment:
I don't think we have such package - did you mean `preset-theme-default?`

Reply:
sorry, thats true